### PR TITLE
[745] Clarify more about SseBrodcaster.onClose() behavior

### PR DESCRIPTION
**Pull request by NicoNes**
Issue: https://github.com/eclipse-ee4j/jaxrs-api/issues/745

### CODE REVIEW

File: jaxrs-api/src/main/java/jakarta/ws/rs/sse/SseBroadcaster.java [hunk 1]
```diff
@@ -36,7 +36,7 @@
 public interface SseBroadcaster extends AutoCloseable {
 
     /**
-     * Register a listener, which will be called when an exception was thrown by a given SSE event output when trying to
+     * Register a listener, which will be called when an exception is thrown by a given SSE event output when tries to
```
Comment (andymc12) [inline jaxrs-api/src/main/java/jakarta/ws/rs/sse/SseBroadcaster.java:39]:
```suggestion
     * Register a listener, which will be called when an exception is thrown by a given {@link SseEventSink} when this SseBroadcaster tries to
```

"when tries" isn't clear who/what is trying.  My suggestion is to use "when this SseBroadcaster tries" instead.

Comment (NicoNes) [reply jaxrs-api/src/main/java/jakarta/ws/rs/sse/SseBroadcaster.java]: @andymc12  you're right it makes more sense like that.

Change done !

Thanks

**Comment by NicoNes** [comment]
@andymc12 you're right it makes more sense like that.

Change done !

Thanks

**Comment by chkal** [comment]
Thanks a lot @NicoNes

**Comment by andymc12** [comment]
This PR has met all the required approvals and no negative votes. Merging on behalf of @NicoNes

**Pull request merged**